assets/src/dashboard/app/views/templateDetails/index.js

```diff
@@ -116,14 +121,33 @@ function TemplateDetails() {
       return;
     }
 
+    if ((!templates || Object.values(templates).length === 0) && !isLoading) {
+      fetchExternalTemplates();
+      return;
+    }
+
+    if (isLoading) {
+      return;
+    }
+
     const id = parseInt(templateId);
     const isLocalTemplate = isLocal && isLocal.toLowerCase() === 'true';
     const templateFetchFn = isLocalTemplate
       ? fetchMyTemplateById
       : fetchExternalTemplateById;
 
-    templateFetchFn(id).then(setTemplate);
-  }, [fetchExternalTemplateById, fetchMyTemplateById, isLocal, templateId]);
+    templateFetchFn(id)
+      .then(setTemplate)
+      .catch(() => {});
```

Comment:
Is there a reason we don't want this error to bubble/throw? Likewise, if it should bubble a helpful message here (https://github.com/google/web-stories-wp/pull/4261/files#diff-b157849a08f1cfe18548cef5b83851c1L241) would be useful

Reply:
Oh yeah! Good call on using a toast. Right now all our templates are local so if it fails we've messed up in the bundling or code.